### PR TITLE
Documentation Updates for Save-In-Progress Routes

### PR DESCRIPTION
**Pull request by edmangimelli**
For PR #5582

***NOTE:*** Some specs added to test the `InProgressFormsController#update` schema

### CODE REVIEW

**Comment by ericboehs** [comment]
Could merge/rebase master in so we can see the actual diff?

**Comment by edmangimelli** [comment]
> Could you merge/rebase master in so we can see the actual diff?

@ericboehs 

it looks right to me 🤷 

let me know if I need to take further action

File: spec/request/in_progress_forms_request_spec.rb [hunk 1]
```diff
@@ -307,6 +306,32 @@ def response_body
...
+              headers: { 'CONTENT_TYPE' => 'application/json' }
+          expect(response).to have_http_status(:ok)
+        end
+
+        it 'can\'t have nil formData' do
+          put v0_in_progress_form_url(new_form.form_id)
+          expect(response).to have_http_status(:error)
+        end
+
+        it 'can\'t have non-hash formData' do
```
Comment (ericboehs) [inline spec/request/in_progress_forms_request_spec.rb:321]: Instead of escaping, could you use `"` instead please?

**Review by ericboehs — APPROVED** [review]
Small request.

File: spec/request/in_progress_forms_request_spec.rb (hunk 1 above)
Comment (edmangimelli) [reply spec/request/in_progress_forms_request_spec.rb]: No prob! For some reason i though our rubocop settings were enforcing single quotes when not interpolating

**Pull request merged**